tooling/cli/metadata.json

```diff
@@ -4,5 +4,5 @@
     "node": ">= 10.0.0"
   },
   "tauri": "1.1.0",
-  "tauri-build": "1.1.4"
+  "tauri-build": "1.1.1"
```

Comment:
current tauri-build is 1.1.0

Reply:
fix a mistake by doing another mistake, godd job me lol

Reply:
relatable